eureka/lib/centroid.py

```diff
@@ -1,163 +1,163 @@
 # $Author: patricio $
```

Comment:
For consistency, maybe we should delete all instances of this outdated method of version control?